pkg/asset-manager-utils/test/AaveATokenAssetManager.test.ts

```diff
@@ -9,104 +9,134 @@ import { bn, fp } from '@balancer-labs/v2-helpers/src/numbers';
 import { MAX_UINT256 } from '@balancer-labs/v2-helpers/src/constants';
 
 import { deploy } from '@balancer-labs/v2-helpers/src/contract';
-import Vault from '@balancer-labs/v2-helpers/src/models/vault/Vault';
 import { expectBalanceChange } from '@balancer-labs/v2-helpers/src/test/tokenBalance';
-import { advanceTime } from '@balancer-labs/v2-helpers/src/time';
 import { encodeJoinWeightedPool } from '@balancer-labs/v2-helpers/src/models/pools/weighted/encoding';
+import { advanceTime } from '@balancer-labs/v2-helpers/src/time';
 
 const OVER_INVESTMENT_REVERT_REASON = 'investment amount exceeds target';
 const UNDER_INVESTMENT_REVERT_REASON = 'withdrawal leaves insufficient balance invested';
 
 const tokenInitialBalance = bn(200e18);
 const amount = bn(100e18);
 
-describe('Aave AToken asset manager', function () {
-  let tokens: TokenList,
-    pool: Contract,
-    vault: Contract,
-    lendingPool: Contract,
-    assetManager: Contract,
-    distributor: Contract,
-    stkAave: Contract;
-  let admin: SignerWithAddress, lp: SignerWithAddress, other: SignerWithAddress;
-  let poolId: string;
+const setup = async () => {
+  const [, admin, lp, other] = await ethers.getSigners();
```

Comment:
I think not having the `setup` function and having this in one (or many) `sharedBeforeEach`s may actually be the better approach (and consistent with all other test files we have). For one, it avoids having to keep the `getSigners()` calls in sync.